namesys/resolve/pathresolver_test.go

```diff
@@ -16,17 +16,17 @@ func TestResolveNoComponents(t *testing.T) {
 	}
 
 	_, err = resolve.Resolve(n.Context(), n.Namesys, n.Resolver, path.Path("/ipns/"))
-	if err != path.ErrNoComponents {
-		t.Fatal("Should error with no components (/ipns/).", err)
+	if err == nil {
```

Comment:
We probably want to check more than 'there was an error'

Reply:
But I'm lazy... fine.